extruct/w3cmicrodata.py

```diff
@@ -46,13 +45,19 @@
 
 class LxmlMicrodataExtractor(object):
     # iterate in document order (used below for fast get_docid)
-    _xp_item = lxml.etree.XPath('descendant-or-self::*[@itemscope]')
-    _xp_prop = lxml.etree.XPath("""set:difference(.//*[@itemprop],
+    _xp_item = lxml.etree.XPath("descendant-or-self::*[@itemscope]")
+    _xp_prop = lxml.etree.XPath(
+        """set:difference(.//*[@itemprop],
                                                   .//*[@itemscope]//*[@itemprop])""",
```

Comment:
that bit looks a bit weird, I wonder if there is a way of editing it so that black is happy and it looks decent?